# This is for python 2.x
def raise_error_trace(err, traceback):
    raise err.__class__, err, traceback